src/extendables/User/Minion.ts

```diff
@@ -258,6 +261,10 @@ export default class extends Extendable {
 			case Activity.FightCaves: {
 				return `${this.minionName} is currently attempting the ${Emoji.AnimatedFireCape} **Fight caves** ${Emoji.TzRekJad}.`;
 			}
+			case Activity.TitheFarm: {
+				return `${this.minionName} is currently attempting the **Tithe Farm**. ${formattedDuration}`;
```

Comment:
"attempting" tithe farm sounds weird, it should be  more like "is currently farming at the Tithe Farm"